mate-session/gsm-logout-dialog.c

```diff
@@ -464,30 +464,30 @@ gsm_get_dialog (GsmDialogLogoutType type,
                 logout_dialog->default_response = GSM_LOGOUT_RESPONSE_SHUTDOWN;
 
                 if (gsm_logout_supports_system_suspend (logout_dialog)) {
-                        gtk_dialog_add_button (GTK_DIALOG (logout_dialog),
-                                               _("S_uspend"),
+                        gsm_util_dialog_add_button (GTK_DIALOG (logout_dialog),
+                                               _("S_uspend"), "system-suspend",
                                                GSM_LOGOUT_RESPONSE_SLEEP);
                 }
 
                 if (gsm_logout_supports_system_hibernate (logout_dialog)) {
-                        gtk_dialog_add_button (GTK_DIALOG (logout_dialog),
-                                               _("_Hibernate"),
+                        gsm_util_dialog_add_button (GTK_DIALOG (logout_dialog),
+                                               _("_Hibernate"), "system-hibernate",
```

Comment:
in my tests, this icon is missing

Reply:
Same as above, updated soon, thanks for all again.

Reply:
Done, switch to another one.